tsdb/store.go

```diff
@@ -6,6 +6,7 @@ import (
 	"context"
 	"errors"
 	"fmt"
+	errors3 "github.com/influxdata/influxdb/v2/pkg/errors"
```

Comment:
`errors3` should be in a separate import block following the standard Go packages.

Reply:
Turns out GoLand was set to `gofmt` import style rather than `goimports`. Seems a violation of Go's whole ethos to have 2 different formatting styles.

Reply:
Been bitten by that many times....